dotnet/dotnet-cloud-run-hello-world/Pages/Index.cshtml

```diff
@@ -36,9 +36,27 @@
     </p>
 
     <h3>What's next?</h3>
-    <p class="next">Manage Cloud APIs and libraries in Cloud Code for <a href="https://cloud.google.com/code/docs/vscode/client-libraries" target="_blank" rel="noopener">VSCode</a> or <a href="https://cloud.google.com/code/docs/intellij/client-libraries" target="_blank" rel="noopener">IntelliJ</a></p>
-    <p class="next">Explore more options with
+    <ul>
+        <li class="next">
+            Want to help shape the future of Cloud Code? Sign up for the chance to participate in our ongoing research studies 
+            <a href="https://google.qualtrics.com/jfe/form/SV_4Me7SiMewdvVYhL?reserved=1&utm_source=In-product&Q_Language=en&utm_medium=own_prd&utm_campaign=Q1&productTag=clou&campaignDate=January2021&referral_code=UXlX318928" target="_blank" rel="noopener">
+            here!
+            </a>
+        </li>
+        <li class="next">Try out live debugging in Cloud Code for <a href="https://cloud.google.com/code/docs/vscode/debugging-a-cloud-run-service" target="_blank" rel="noopener">VS Code</a> or <a href="https://cloud.google.com/code/docs/intellij/debugger" target="_blank" rel="noopener">IntelliJ</a></li>
+        <li class="next">Extend your Cloud Code application with Secret Manager for <a href="https://cloud.google.com/code/docs/vscode/secret-manager" target="_blank" rel="noopener">VS Code</a> or <a href="https://cloud.google.com/code/docs/intellij/secret-manager" target="_blank" rel="noopener">IntelliJ</a></li>
+        <li class="next">Joing the <b>#cloud-code</b> channel in the <a href="https://join.slack.com/t/googlecloud-community/shared_invite/zt-erdf4ity-8ZMUQ18DYV~5hkbZ~gCswg" target="_blank" rel="noopener">Google Cloud Platform Slack community</a> to ask questions, provide feedback, and get the latest Cloud Code news</li>

```

Comment:
Misspell, also in other sample pages html.

Reply:
Whoops, thanks for catching! Fixed